test/uk/gov/hmrc/gform/sharedmodel/formtemplate/generators/FormatExprGen.scala

```diff
@@ -16,7 +16,7 @@
 
 package uk.gov.hmrc.gform.sharedmodel.formtemplate.generators
 import org.scalacheck.Gen
-import uk.gov.hmrc.gform.sharedmodel.formtemplate.{ After, AnyDate, AnyText, AnyWord, BasicText, Before, BeforeOrAfter, ConcreteDate, CountryCode, DateConstraint, DateConstraintInfo, DateConstraintType, DateConstraints, DateField, Email, NINO, NextDate, NonUkCountryCode, Number, OffsetDate, PositiveNumber, PreviousDate, ShortText, Sterling, TelephoneNumber, TextConstraint, TextExpression, TextWithRestrictions, Today, UTR, UkBankAccountNumber, UkSortCodeFormat, UkVrn }
+import uk.gov.hmrc.gform.sharedmodel.formtemplate.{After, AnyDate, AnyText, AnyWord, BasicText, Before, BeforeOrAfter, CompanyRegistrationNumber, ConcreteDate, CountryCode, DateConstraint, DateConstraintInfo, DateConstraintType, DateConstraints, DateField, Email, NINO, NextDate, NonUkCountryCode, Number, OffsetDate, PositiveNumber, PreviousDate, ShortText, Sterling, TelephoneNumber, TextConstraint, TextExpression, TextWithRestrictions, Today, UTR, UkBankAccountNumber, UkSortCodeFormat, UkVrn}
```

Comment:
On gform we use automatic code formatter (scalafmt) to alleviate us from problem of discussing/agreeing on one coding style. This line has not been formatted by it though. 
To get it formatted simplest way is to run command `test:compile` in sbt.
Another way may be to invoke scalafmt as a post save action: https://scalameta.org/scalafmt/docs/installation.html from IDE.

Reply:
<img width="522" alt="image" src="https://user-images.githubusercontent.com/4254983/50780382-b310d880-129a-11e9-9eab-66c42168d697.png">

You can make IntelliJ use scalafmt from Preferences too